frontend/common/package.json

```diff
@@ -24,9 +25,9 @@
     "check-intl-admin-merge": "node ./src/tooling/checkIntl.js --en ../admin/src/js/lang/en.json --fr ../admin/src/js/lang/fr.json --rm-orphaned --output-untranslated ../admin/src/js/lang/untranslated.json --whitelist ../admin/src/js/lang/whitelist.yml --merge-fr ../admin/src/js/lang/newTranslations.json",
     "check-intl-talentsearch": "node ./src/tooling/checkIntl.js --en ../talentsearch/src/js/lang/en.json --fr ../talentsearch/src/js/lang/fr.json --rm-orphaned --output-untranslated ../talentsearch/src/js/lang/untranslated.json --whitelist ../talentsearch/src/js/lang/whitelist.yml",
     "check-intl-talentsearch-merge": "node ./src/tooling/checkIntl.js --en ../talentsearch/src/js/lang/en.json --fr ../talentsearch/src/js/lang/fr.json --rm-orphaned --output-untranslated ../talentsearch/src/js/lang/untranslated.json --whitelist ../talentsearch/src/js/lang/whitelist.yml --merge-fr ../talentsearch/src/js/lang/newTranslations.json",
-    "check-intl-indigenousapprenticeship": "node ./src/tooling/checkIntl.js --en ../indigenousapprenticeship/src/js/lang/en.json --fr ../indigenousapprenticeship/src/js/lang/fr.json --rm-orphaned --output-untranslated ../indigenousapprenticeship/src/js/lang/untranslated.json --whitelist ../indigenousapprenticeship/src/js/lang/whitelist.yml",
-    "check-intl-indigenousapprenticeship-merge": "node ./src/tooling/checkIntl.js --en ../indigenousapprenticeship/src/js/lang/en.json --fr ../indigenousapprenticeship/src/js/lang/fr.json --rm-orphaned --output-untranslated ../indigenousapprenticeship/src/js/lang/untranslated.json --whitelist ../indigenousapprenticeship/src/js/lang/whitelist.yml --merge-fr ../indigenousapprenticeship/src/js/lang/newTranslations.json",
-    "check-integrity": "node ./src/tooling/checkIntegrity.js --lock-file ../package-lock.json --allow common admin talentsearch indigenousapprenticeship"
+    "check-intl-web": "node ./src/tooling/checkIntl.js --en ../../apps/web/src/lang/en.json --fr ../../apps/web/src/lang/fr.json --rm-orphaned --output-untranslated ../../apps/web/src/lang/untranslated.json --whitelist ../../apps/web/src/lang/whitelist.yml",
```

Comment:
How should the translation scripts be run now?  Just run `check-intl-admin` from the common directory as usual?  Seems to work OK.  Do you have thoughts about where those should live in the new directory structure?

Reply:
Yeah, so for now they can be run with `npm run check-intl-admin --workspace=frontend/common`.

I see these living in something like `packages/check-intl` and possibly add a binary to be run directly. Otherwise, it would be as simple as adding something like the following to scripts:

```json
{
  "scripts": {
     "check-intl-admin": "turbo run check-intl-admin --filter=@gc-digital-talent/check-intl"
  }
}
```